packages/expo-updates/ios/EXUpdates/Events/NoOpUpdatesEventManager.swift

```diff
@@ -0,0 +1,18 @@
+//  Copyright © 2019 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+public class NoOpUpdatesEventManager: UpdatesEventManager {
+  public private(set) var appContext: AppContext?
+  public private(set) var shouldEmitJsEvents: Bool = false
+
+  public func setAppContext(appContext: AppContext?) {
+    self.appContext = appContext
+  }
+
+  public func setShouldEmitJsEvents(shouldEmitJsEvents: Bool) {
+    self.shouldEmitJsEvents = shouldEmitJsEvents
```

Comment:
If this is a "no-op" event manager, maybe `setShouldEmitJsEvents` should do nothing (the value should always be false)?

Reply:
I think it should probably still function since these have getters (though unused currently) that should return the last-set value. The no-op is just that no events are sent.